io/src/librealsense/librealsense_device_manager.cpp

```diff
@@ -120,7 +119,7 @@ void
 pcl::io::librealsense::LibRealSenseDeviceManager::releaseDevice (const std::string& sn)
 {
   boost::mutex::scoped_lock lock (mutex_);
-  pcl::io::librealsense::LibRealSenseDeviceManager::stopDevice (sn);
+  if (!thread_close_flag_[sn]) pcl::io::librealsense::LibRealSenseDeviceManager::stopDevice (sn);
```

Comment:
However, the only caller of the `releaseDevice` is the `LibRealSenseGrabberImpl::~LibRealSenseGrabberImpl` where it calls `stop` first.

The underlying librealsense doesn't provide the `release` method, what's the purpose of the `LibRealSenseDeviceManager::releaseDevice`? Only to remove the mapping?